documentation/configuration.markdown

```diff
@@ -75,9 +79,9 @@ This file contains the settings for Puppet Server itself.
 # configuration for the JRuby interpreters
 
 jruby-puppet: {
-    gem-home: /var/lib/puppet/jruby-gems
+    gem-home: /opt/puppetlabs/puppet/cache/jruby-gems
```

Comment:
The AIO puppet-agent doesn't have a GEM_HOME like puppet-server does, so this value comes from the previous behavior of puppet-server placing GEM_HOME into `$vardir/gem-home`.